src/index.html

```diff
@@ -5,8 +5,405 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="navigation">
+        <a href="#home" class="navigation__logo">
+          <img src="images/logo.png" alt="logo">
+        </a>
+
+        <nav class="nav">
+          <ul class="menu">
+            <li class="menu__category">
+              <a class="menu__menuLink is-active" href="#">Apple</a>
```

Comment:
don't use the camel case for classes
```suggestion
              <a class="menu__menu-link is-active" href="#">Apple</a>
```